docs/updating.md

```diff
@@ -0,0 +1,69 @@
+# Upgrading
+
+BACK UP EVEYTHING. The database, the mounted volumes, everything. Please. Pretty please.
+
+While this process (almost) always works there can be errors and we really don't want you to loose any data.
+
+## The apache image
+
+Just change the tag name and restart the image.
+
+## The fpm image
+
+The FPM image will need to be upgraded with a manual step. Because the FPM image will have a HTTP proxy (normally nginx) container serving the static assets the ```public``` directory is mounted into that nginx container. This is done via volumes:
+
+```yaml
+version: '3.5'
+services:
+    kimai:
+        image: kimai/kimai2
+        ...
+        volumes:
+          - public:/opt/kimai/public
+        ...
+    nginx:
+        ...
+        volumes:
+            - public:/opt/kimai/public:ro
+    ...
+```
+
+When the kimai image is updated and the container is restarted any new assets in the public directory are never included. These will be things like CSS files, images etc. To fix this you need to copy the newer files from a fresh image oved the top.
+
+```bash
+me@myhost $ docker run --rm -ti -v kimai_public:/public --entrypoint /bin/bash kimai/kimai2
```

Comment:
isn't that `public` instead of `kimai_public` everywhere else?
I mean it is good to make people think about what they do, [but here it is `public` as well](https://github.com/tobybatch/kimai2/blob/main/docker-compose.yml#L48)

Reply:
Ah Ha! It's one of those docker things ;)

When start a docker-compose stack the volume name is prefixed with the name of the folder the docker-compose.yml file is in.  There may be an argument for this being kimai2_public.

Reply:
LOL, one time I do not mention "I don't know docker" and then that happens 😁